com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkConstants.cs

```diff
@@ -0,0 +1,66 @@
+namespace MLAPI.Configuration
+{
+    /// <summary>
+    /// A static class containing MLAPI constants
+    /// </summary>
+    internal static class NetworkConstants
+    {
+        internal const string k_PROTOCOL_VERSION = "13.0.0";
```

Comment:
Is this a Unity convention?  Microsoft says just use camel case for constants.  The k_ prefix seems a bit unexpected

Reply:
yep, I base everything on Unity standards (Unity's `CSharpReference.cs`).

Reply:
Ok, fair enough, but the doc says use the prefixes for non-public elements.  I take that to mean things you only see within the class they're defined.  We see these comments all around the code (e.g. k_ADD_OBJECT in RpcQueueProcessor.cs)

Reply:
hmm, I think treating `internal` modifier as `public` would make sense and I do agree on reinterpreting "non-public" as things that are not visible from the outside of the class itself. in this case, `internal` is in fact visible so it's `public` in that regard.

I will apply this logic to other places as well — I will update things.